graql/language/define.feature

```diff
@@ -983,14 +989,9 @@ Feature: Graql Define Query
 
 
   # TODO
-  Scenario Outline: a type can own a '<value_type>' attribute type as a key
-
-    Examples:
-
-  # TODO
-  Scenario Outline: a '<value_type>' attribute type is not allowed to be a key
-
-    Examples:
+#  Scenario Outline: a type can own a '<value_type>' attribute type as a key
+#
+#  Scenario Outline: a '<value_type>' attribute type is not allowed to be a key
```

Comment:
The reason Client Python's `behave` framework hated these scenarios (and threw a fatal error) was because of their empty `Examples` blocks. Given that it's syntactically invalid Cucumber to have a `Scenario Outline` with no `Examples` block, the only option that satisfies all our BDD frameworks is to comment them out for now.